recipe/conda_build_config.yaml

```diff
@@ -2,6 +2,6 @@ mpi:
   - mpich
   - openmpi
 c_compiler_version:
-  - '16'  # [osx]
+  - '13'  # [linux]
 cxx_compiler_version:
-  - '16'  # [osx]
+  - '13'  # [linux]
```

Comment:
Why is it necessary to now add pinning for linux builds?

Reply:
Because we were using GCC 13 as the default compiler before, and the recent switch to GCC 14 broke the build here. This keeps the compiler version inline with what this feedstock has been successfully been using over the last months.